src/models/layerMetadata/vectorBestMetadata.ts

```diff
@@ -18,7 +18,7 @@ export interface IVectorBestMetadata {
   productVersion: string | undefined;
   // resolution: number | undefined;
   // rms: number | undefined;
-  scale: number | undefined;
```

Comment:
scale should be a string or a number?

Reply:
we didn't won't to break other types compatibility - so we changed only our data